cachi2/core/models/input.py

```diff
@@ -216,6 +216,7 @@ class RpmPackageInput(_PackageInputBase):
     """Accepted input for a rpm package."""
 
     type: Literal["rpm"]
+    add_rpm_summary: bool = False
```

Comment:
what is the reasoning behind making this configurable? On its own the option doesn't IMO make much sense and I'd like to avoid littering the code with semi-random single-purpose tailored SBOM options in the future by setting this precedent. It's a straightforward change, so why don't we make it the default?